bootstrap.sh

```diff
@@ -24,14 +24,18 @@ brew cask install fluid
 brew cask install 1password
 brew cask install quicksilver
 
-brew install maven
+# Install various utilities through homebrew
 brew install ant
+brew install git
```

Comment:
Not sure whether git was missing from the script on purpose. I suppose you won't get at this repo without having git installed first, but it should be okay to include it.